server-function/shift_deploy_binaris.sh

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+rm -rf .shift_deploy
+npm run build
+mkdir -p .shift_deploy/node_modules
+cp -a build .shift_deploy
+for i in $(node getdeps.js) ; do
+  dirtocreate=$(dirname node_modules/$i)
+  mkdir -p .shift_deploy/$dirtocreate
+  cp -a node_modules/$i .shift_deploy/$dirtocreate
+done
+cp -a `ls | grep -v node_modules | grep -v backend` .shift_deploy
+cat >.shift_deploy/binaris.yml <<EOF
+functions:
+  public_shift_builtin:
+    file: function.js
+    entrypoint: handler
+    runtime: node8
+EOF
+cat >.shift_deploy/function.js <<EOF
```

Comment:
why embed it in the shell script?
wouldn't keeping it separate make development easier (editors) and allow unit testing?